src/assets/scss/_variables.scss

```diff
@@ -98,6 +100,25 @@ $xlg: 2.5rem; // 40px
 $xxlg: 3.15rem; // 50px
 $xxxlg: 3.75rem; // 60px
 
+// Breakpoints.
+
+$break-xxsm: 400px;
+$break-xsm: 600px;
+$break-sm: 768px;
+$break-md: 980px;
+$break-lg: 1200px;
+$break-page-container: 84.375em;
+$break-xlg: 106em;
 
 // Drop shadow.
+
 $soft-drop-shadow: 0px 1px 2px rgba(0, 0, 0, 0.4);
+
+
+:root {
```

Comment:
We should probably move these to `base.scss` so they don't load on every component - correct?